MIP12/mip12.md

```diff
@@ -6,9 +6,10 @@ MIP#: 12
 Title: Collateral and Risk Parameter Management
 Author(s): Charles St.Louis (@CPSTL), Rune Christensen (@Rune23) 
 Type: Technical, Process
-Status: Request for Comments (RFC)
+Status: Accepted
```

Comment:
I think it's bad practice to change the status and make changes in the same PR. Please can we avoid this?

Reply:
Agreed. Note that MIP6 is already set to the 'Accepted' status in the master/Accepted branches and this change is just reflected because the RFC branch was not as up-to-date as the aforementioned branches.